san-francisco-tourism/build.gradle

```diff
@@ -0,0 +1,33 @@
+
+group = 'com.oracle.javaone.2017.hol.junit5'
+version = '0.0.1-SNAPSHOT'
+
+description = "junit5 with gradle as build system"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+ext.junitVintageVersion  = '5.0.0'
```

Comment:
are these dependencies for the solution, rather than the exercise (e.g. where we want to get to)?